db/seeds.dev.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+FactoryBot.create_list(:account, 3)
+
+Account.find_each do |account|
+  host_ids = SupportedSsg.all.map { |s| [s.os_major_version, s.os_minor_version] }.uniq.flat_map do |version|
+    FactoryBot.create_list(:host, rand(2..3), account: account.account_number, os_version_arr: version).map(&:id)
+  end
+
+  Profile.canonical.includes(:benchmark).group_by(&:ref_id).each do |_, profiles|
+    canonical_profile = profiles.sample
+    hosts = Host.where(id: host_ids).where("#{Host::OS_MAJOR_VERSION} = ?", canonical_profile.os_major_version)
+
+    next if hosts.empty? || [true, true, false].sample
+
+    profile = Profile.new(parent_profile_id: canonical_profile.id, account: account).fill_from_parent
+    policy = Policy.new(profile.slice(:name, :description, :compliance_threshold, :account))
+    policy.save!
+    profile.update!(policy: policy)
+
+    policy.update_hosts(hosts.pluck(:id))
+
+    passing_hosts = policy.hosts.sample(rand(1..policy.hosts.count / 2))
+
+    hosts.find_each do |host|
+      next if [true, true, false].sample
```

Comment:
...curious, is this to randomly skip some hosts?

Reply:
Yup, up to 30% so you have the opportunity to assign hosts on your own through the UI/API